src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/SupportController.java

```diff
@@ -30,6 +31,7 @@
 
 @RestController
 @RequestMapping("/support")
+@ConditionalOnProperty(prefix = "feature_toggles", name = "emailToStaff")
```

Comment:
are you sure you want to disable the whole controller when you don't want to send emails to staff?

It looks a bit too nuke for me :)

Reply:
I agree it does seem a bit overkill.
Im open to a better solution

Reply:
there is no solution, bean it contains get disabled. so it need to disabled otherwise will give spring autowiring error